repository_service_tuf_worker/services/keyvault/local.py

```diff
@@ -20,31 +20,63 @@ class KeyVaultError(Exception):
 class LocalKeyVault(IKeyVault):
     """Local KeyVault type"""
 
-    def __init__(self, path: str):
+    def __init__(
+        self,
+        path: str,
+        online_key_name: Optional[str] = "online.key",
+        online_key_pass: Optional[str] = None,
+        online_key_type: Optional[str] = "ed25519",
+    ):
+        """A configuration class for RSTUF Worker services.
+
+        Args:
+            path: directory of the key vault.
+            online_key_name: file name of the online key.
+            online_key_pass: password to load the online key.
```

Comment:
Document the supported list from the enum. (see my comment above)

Reply:
See my comment above about the supported algorithms.